documentation/book/con-kafka-connect-authentication.adoc

```diff
@@ -74,10 +74,13 @@ spec:
 To configure Kafka Connect to use SASL based PLAIN authentication, set the `type` property to `plain`.
 This authentication mechanism requires a username and password.
 
+WARNING: SASL PLAIN mechanism transfers the username and password in cleartext over the network.
+It should not be used without TLS encryption.
```

Comment:
I would say in the other way around, "it should be used only with TLS encryption"